README.md

```diff
@@ -24,17 +38,14 @@ This project requires Java 8 and can be built with [Gradle](https://gradle.org).
 * Simon Fraser University
 * PALS
 
-## Maintainers
-
-* [Nick Ruest](https://github.com/ruebot)
-* [Daniel Lamb](https://github.com/dannylamb/)
-
 ## Development
 
 If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
 
 If you would like to contribute code to the project, you need to be covered by an Islandora Foundation [Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_cla.pdf) or [Corporate Contributor Licencse Agreement](http://islandora.ca/sites/default/files/islandora_ccla.pdf). Please see the [Contributors](http://islandora.ca/resources/contributors) pages on Islandora.ca for more information.
 
+We recommend using the [claw-playbook](https://github.com/Islandora-Devops/claw-playbook) to get started. 
+
 ## Licensing
 
-MIT
+[MIT](https://github.com/Islandora-CLAW/Alpaca/pull/LICENSE)
```

Comment:
Sorry again! I should have escaped my suggested text. It's `[MIT](/License)` to make the link work.